samples/Directory.Build.props

```diff
@@ -12,5 +12,26 @@
     <IsPackable>false</IsPackable>
     <MainLibraryPath>$(MSBuildThisFileDirectory)../src/System.Device.Gpio/</MainLibraryPath>
     <IotBindingsLibraryPath>$(MSBuildThisFileDirectory)../src/Iot.Device.Bindings/</IotBindingsLibraryPath>
+    <PreferPackageReference></PreferPackageReference>
```

Comment:
```suggestion
```

I meant you can just remove it entirely. When you don't have it defined it evaluates to empty, for example if you were to use `'$(SomeRandomPropertyThatDoesNotGetDefinedAnywhereAtAll)'` will evaluate to `''`